lang/octave/README.md

```diff
@@ -0,0 +1,271 @@
+dependencies. 
+
+If you need to install additional Octave toolboxes, you can do so with `apt`.  
+For instance, if you need the Octave signal processing  toolbox, add the 
+following line to the `%post` section of the definition file.
+
+```
+apt-get install -y octave-signal
+```
+
+The `%environment` section defines variables that will be set during runtime. In
+this case, we need to set the `LC_ALL` variable so that Octave doesn't complain
+about the locale being unset.
+
+Finally, the `%runscript` section defines the commands that the container will 
+execute when it is called either with `singularity run` or called by name as if 
+it were an executable.
+
+## building the container
+
+After writing or downloading the Octave definition file, you can build the 
+container with the following command:
+
+```
+$ sudo singularity build octave.sif octave.def
+```
+
+What if you want to build a container, but you don't have administrative access
+(root) in your Linux environment? You can use the
+[remote builder](https://cloud.sylabs.io/builder) (assuming your environment has
+access to the web) to create your container like so:
+
+```
+$ singularity build --remote octave.sif octave.def
+```
+
+If it is your first time using the Remote Builder, follow 
+[these instructions](https://cloud.sylabs.io/auth) to create an identification 
+token. Your container will be built on the Remote Builder and all standard out
+and standard err will be directed back to your terminal. When finished, the 
+container will be automatically downloaded to your local machine.
+
+What if you don't have Singularity installed or you don't have a Linux 
+environment?  After signing in to the Sylabs Cloud you can compose definition 
+files directly in the window on the main 
+[Remote Builder](https://cloud.sylabs.io/builder) page or drag and drop text 
+files there. The containers will show up under your username in the 
+`remote-builds` collection. Then you can download it with the `singularity pull`
+command.
+
+## running the container
+
+After using one of the methods above to build your container, you can run it 
+several different ways.  
+
+### starting the Octave IDE
+
+If you want to start an interactive development session you can either invoke
+the container using the `singularity run` command, or you can treat the 
+container like an executable and start it like so:
+
+```
+$ ./octave
+```
+
+The runscript within the container will execute the command `octave "$@"`. Since
+no additional arguments were supplied, this is the same as simply executing the
+command `octave` within the container, which will begin an interactive 
+programming session. (Note that this assumes your environment can support a 
+graphical user interface.)
+
+### running a script through your octave container
+
+If you would rather use your Octave container to execute a script, you can pass 
+it as an argument. In this repo, we are providing the following example script.
+
+```matlab
```

Comment:
Whats `matlab` doing there? I dont think its nessary.

Reply:
It's for syntax highlighting

Reply:
:+1: